usr/lib/xtable/libxtable.c

```diff
@@ -0,0 +1,216 @@
+/*
+* SPDX-FileCopyrightText: (c) 2024 Mohammad Shehar Yaar Tausif <sheharyaar48@gmail.com>
+* SPDX-License-Identifier: MIT OR GPL-2.0-only
+*/
+
+#include <lauxlib.h>
+#include <lua.h>
+#include <lualib.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <xtables.h>
+
+#include "luaxtable.h"
+
+#ifndef LUAXTABLE_MODULE
+#define LUAXTABLE_MODULE "test"
+#endif
+
+static lua_State *L = NULL;
+const char *libxtable_match_key = "libxtable_match";
+const char *libxtable_target_key = "libxtable_target";
+
+int luaopen_libxtable(lua_State *L);
+
+// Function to initialize Lua state
```

Comment:
I would get rid of this sort of comment.. you don't need to "teach" Lua through them..